lib/shrine/plugins/determine_mime_type.rb

```diff
@@ -179,9 +179,14 @@ def extract_with_file(io)
 
             raise Error, "file command failed to spawn: #{stderr.read}" if status.nil?
             raise Error, "file command failed: #{stderr.read}" unless status.success?
+
             $stderr.print(stderr.read)
 
-            stdout.read.strip
+            output = stdout.read.strip
+
+            raise Error, "file command failed: #{output}" if output.start_with?("cannot open")
```

Comment:
Will this strict start with matching "cannot open" work across platforms/OS or would it better to match "cannot open" or "no file" anywhere in the output and downcase output?

Reply:
Yeah, we don't have to match at the start, maybe that will match more platforms. Between "cannot open" and "No such file", I don't know which one would be more cross-platform, since I don't know how messages will differ 🤷‍♂️